Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt

```diff
@@ -168,6 +164,10 @@ class CardFragment : Fragment() {
         }
     }
 
+    private fun showTestCards() {
+        findNavController().navigate(R.id.action_cardFragment_to_testCardFragment)
```

Comment:
nit: since we've enabled viewBinding, directions can be referred from `CardFragmentDirections` instead of using `r.id`